intelipost.gemspec

```diff
@@ -18,8 +18,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'faraday', '~> 0.9.1'
-  spec.add_dependency 'faraday_middleware', '~> 0.9.1'
+  spec.add_dependency 'faraday', '>= 0.11.0'
```

Comment:
Since Faraday is semantically versioned, I recommend using `~> 0.11.0` syntax to prevent future braking updates.